2019/media/playbackperfTest.js

```diff
@@ -25,7 +25,17 @@ var PlaybackperfTest = function() {
 
 var webkitPrefix = MediaSource.prototype.version.indexOf('webkit') >= 0;
 var tests = [];
-TestBase.timeout = 70000; // 70 sec to compensate for 0.25 playbackRate tests.
+var testTime = harnessConfig.testTime;
+
+if (harnessConfig.timeout == TestBase.timeout) {
+  // Use a timeout (in ms) that is 4x highger than testTime
+  // to compensate for 0.25 playbackRate tests.
+  TestBase.timeout = harnessConfig.testTime * 4000;
```

Comment:
Given the preview test time of 15, the timeout appears to have an additional 10 second padding.